### DISCUSSION TITLE
请教一下私密日记的评论和其他奇怪问题

### DISCUSSION THREAD

**Discussion by zdltvxq** [ShunCai/QZoneExport#65] [Q&A] [answered]
你好，
我有两个小问题
1、有些说说，设置成了仅自己可见了，然后在空间主页显示，但是在说说页面里找不到（可能是QQ空间的BUG），这样的日志没下载下来
2、私密日记里的评论没有备份下来，因为有些私密日记是以前的日志转成私密的，这部分数据应该也能拿到的吧？或者麻烦告知一下哪段代码或者哪个接口可以有？
3、相册里的图有些是有配文字的，也没有备份下来

**Comment by zdltvxq** [comment, marked as answer]
FK，是三个问题

**Reply by ShunCai** [reply]
问题一：我这边没遇到过，我可以尝试下，不过如果确认说说页面不能找到，那确实备份不下来，因为助手就是基于说说列表清单接口进行备份的。

问题二：私密日志的评论，助手好像就直接没备份，因为当初没想到私密日志也会有评论，理论上跟日志评论接口应该是公用的，我得先瞧瞧。

问题三：文字有备份下来的，只是当初没显示出来，后面会优化下。
有编码能力的，也可以自行修改，比如修改QQ空间备份/Albums/photos.html文件
如下所示，就可以优先显示相片描述，描述为空再显示名称，或者合并展示，
[![hdy7bn.png](https://z3.ax1x.com/2021/08/31/hdy7bn.png)](https://imgtu.com/i/hdy7bn)

**Reply by ShunCai** [reply]
问题二和问题三，dev分支改了，可以自己下载下了离线安装。

**Comment by zdltvxq** [comment]
多谢。有空我看看



发自我的iPhone


------------------ 原始邮件 ------------------
发件人: 芷炫 ***@***.***&gt;
发送时间: 2021年8月31日 22:08
收件人: ShunCai/QZoneExport ***@***.***&gt;
抄送: 黑主理事长 ***@***.***&gt;, Author ***@***.***&gt;
主题: 回复：[ShunCai/QZoneExport] 请教一下私密日记的评论和其他奇怪问题 (#65)





 
问题一：我这边没遇到过，我可以尝试下，不过如果确认说说页面不能找到，那确实备份不下来，因为助手就是基于说说列表清单接口进行备份的。
 
问题二：私密日志的评论，助手好像就直接没备份，因为当初没想到私密日志也会有评论，理论上跟日志评论接口应该是公用的，我得先瞧瞧。
 
问题三：文字有备份下来的，只是当初没显示出来，后面会优化下。
 有编码能力的，也可以自行修改，比如修改QQ空间备份/Albums/photos.html文件
 如下所示，就可以优先显示相片描述，描述为空再显示名称，或者合并展示，
 
 
—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub, or unsubscribe.
Triage notifications on the go with GitHub Mobile for iOS or Android.
